### DISCUSSION TITLE
Proposal: "Core" team meetings for Pioneers and Contributors (evolution of "Weekly All Team")

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#431] [Ideas]
Proposal

- **What**: A "core" team meeting focused on presencing what we are up to ([SCQH](https://lifeitself.org/tao/scqh) etc), strategic direction and updates
- **Who**: Elders, Pionners and Contributors can attend (plus any paid collaborators) (nb: all hub-holders are contributors or pioneers)
- **When**: biweekly (probably on friday afternoon european time) **UPDATE: likely either 1330 European or 1630 european for ~1h**

Situation

- Have had regular-ish weekly or biweekly meetings over last few years "Weekly all team"
- Attendees were a mix of paid collaborators, founders and pioneers
- Recent evolution e.g. new Bergerac Hub holders

Complication

- Not clear who attends
- Meeting has fluctuating attendance
- Unclear purpose to meeting (e.g. is it check-in, strategic discussion, SCQH etc)

Question: what meeting should we have with whom at what regularity?

Hypothesis: see the proposal

**Comment by msantolini** [comment]
Excellent idea. Would be happy to join such a strategic meeting to sync on directions. Friday around the LI research meeting would be great, as it would allow to pull together the time investment.

**Reply by rufuspollock** [reply]
👍 and agree re consolidating the time chunk for folks!

**Comment by Liyubov** [comment]
Hey, 
I agree, it is good to do a global sync up call.

To add to the **situation**: 

- There many people who have been part of the **residencies** in the Hub (Bergerac), and would like to contribute but still do not know how and in which role
- It people are researchers in academia (or tech) it is easy to understand how they can contribute to Research
- Currently there are **whatsapp** groups for France, but they are not really used for specific goals, but maybe we could more direct them for people syncronising how they could contribute?


Thanks.

**Reply by rufuspollock** [reply]
> There many people who have been part of the residencies in the Hub (Bergerac), and would like to contribute but still do not know how and in which role

@Liyubov having experience with volunteers offers over 20y now, when people say they want to contribute the first thing i would ask them is to post an introduction here in the forum. And ask that in addition to their standard intro please also :

- Confirm they want to contribute and why
- How much time they can commit per month over the next X months (e.g. next 6 months)
- Any prior experience contributing to a project on a voluntary basis

**Reply by Liyubov** [reply]
thanks @rufuspollock  
this is smth we discussed with @msantolini 

generally we have now feeling that best way not to loose energy on constructing monstrous alumni network is indeed:

- use dyadic connections (one-on-one) mentor like network when people want and have time to contribute
- invite them to regular community calls (like Thursday general) and monthly research (if they can contribute and want to move)
- collect info and track progress (ideally on github) but issue here is that not all people are usign github

**Comment by nathenf** [comment]
I would love to attend to! I will continue to be on PST until June 7th thought

**Reply by rufuspollock** [reply]
Yes, i hoped you'd make it. I think this would either be around 2pm or 4pm on Friday which may then work for you ...

**Comment by catherinet1** [comment]
Have we set these up? Shall I create a calendar invite for them starting from e.g. next Friday, 4-5pm CET,  fortnightly, @rufuspollock ? I think it would be great to have some time and a rhythm for syncing up, connecting, and seeing where we are at as a whole

**Reply by rufuspollock** [reply]
I sent a first invite for next week on Friday at 2pm European.
